app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
   validates :First_name, :Last_name, presence: true, length: {minimum:3 , maximum:50}
```

Comment:
In rails we use snake_case which means all lowercase so:
`First_name` should be `first_name`
`Last_name ` should be `last_name `
`Year_of_birth ` should be `year_of_birth `